packages/core/src/command/consolidated/search/fhir-resource/__tests__/search-consolidated.test.ts

```diff
@@ -121,41 +121,6 @@ describe("search-consolidated", () => {
       });
     });
 
-    describe("non-specialized hydration", () => {
-      runTest(nonSpecializedHydration.conditionAndEncounter);
-      runTest(nonSpecializedHydration.conditionAndObservation);
-      runTest(nonSpecializedHydration.encounterAndObservation);
```

Comment:
Shouldn't we just remove the last one, or first and last? I think we didn't touch anything related to condition and observation, eh?